pipelines/_ingredients/ApPipe.yaml

```diff
@@ -120,6 +121,7 @@ subsets:
       - analyzeAssocDiaSrcCore
       - analyzeTrailedDiaSrcCore
       - diffimTaskCore
+      - associationCore
```

Comment:
Can the management confirm this is something we want in `prompt` and not `afterburner`?

Reply:
Yes, this is lightweight and is intended to be run in Prompt Processing so that these metrics can show up in Chronograf.